input_parser.star

```diff
@@ -50,7 +50,7 @@ DEFAULT_IMAGES = {
     "zkevm_bridge_proxy_image": "haproxy:3.1-bookworm",  # https://hub.docker.com/_/haproxy/tags
     "zkevm_bridge_service_image": "hermeznetwork/zkevm-bridge-service:v0.6.2-RC2",  # https://hub.docker.com/r/hermeznetwork/zkevm-bridge-service/tags
     "zkevm_bridge_ui_image": "leovct/zkevm-bridge-ui:multi-network",  # https://hub.docker.com/r/leovct/zkevm-bridge-ui/tags
-    "zkevm_contracts_image": "xavierpolygon/agglayer-contracts:v11.0.0-rc.1-fork.12",
+    "zkevm_contracts_image": "jhkimqd/agglayer-contracts:v11.0.0-rc.2-fork.12",
```

Comment:
Can you update occurrences of `xavierpolygon/agglayer-contracts:v11.0.0-rc.1-fork.1` in `.github/tests/` also? thxx